vision_datasets/common/data_manifest/data_manifest.py

```diff
@@ -199,3 +199,82 @@ def __eq__(self, other) -> bool:
 
     def __len__(self):
         return len(self.images)
+
+
+class AnnotationDataManifest(ManifestBase):
```

Comment:
`AnnotationDataManifest` may be confusing because it sounds like a parallel concept to `ImageDataManifest`. Maybe we should consider something like `MultiImageLabelManifest`?

Reply:
good suggestion, done